app/frontend/src/components/forms/AddTeamMember.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="new-member-div mb-5">
+    <div v-if="addingUsers" class="d-flex justify-end">
+      <v-autocomplete
+        label="Type  a name, IDIR, or email to add member"
+        hide-details
+        outlined
+        dense
+        clearable
+        v-model="model"
+        :items="items"
+        :search-input.sync="searchUsers"
+        :filter="filterObject"
+        return-object
+        :loading="isLoading"
+      >
+        <!-- selected user -->
+        <template v-slot:selection="data">
+          <span
+            v-bind="data.attrs"
+            :input-value="data.selected"
+            close
+            @click="data.select"
+            @click:close="remove(data.item)"
+          >{{ data.item.fullName }}</span>
+        </template>
+        <!-- users found in dropdown -->
+        <template v-slot:item="data">
+          <template v-if="typeof data.item !== 'object'">
+            <v-list-item-content v-text="data.item"></v-list-item-content>
+          </template>
+          <template v-else>
+            <v-list-item-content>
+              <v-list-item-title v-html="data.item.fullName"></v-list-item-title>
+              <v-list-item-subtitle v-html="data.item.username"></v-list-item-subtitle>
+              <v-list-item-subtitle v-html="data.item.email"></v-list-item-subtitle>
+            </v-list-item-content>
+          </template>
+        </template>
+      </v-autocomplete>
+      <!-- buttons -->
+      <v-btn color="primary" class="ml-2" :disabled="!model" :loading="isLoading" @click="save">
+        <span>Add</span>
+      </v-btn>
+      <v-btn outlined class="ml-2" @click="addingUsers = false">
+        <span>Cancel</span>
+      </v-btn>
+    </div>
+
+    <div v-else class="d-flex justify-md-end">
+      <v-btn @click="addingUsers = true" color="primary">Add a New Member</v-btn>
```

Comment:
This is missing a span around the Add a New Member text. Button text is not bold like everything else as a result.
![add button](https://user-images.githubusercontent.com/1546326/98580846-7dadeb00-2275-11eb-9c9f-ef56d509a98c.PNG)